tests/rptest/tests/cluster_features_test.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from rptest.services.admin import Admin
+from rptest.tests.redpanda_test import RedpandaTest
+from ducktape.mark.resource import cluster
+
+from ducktape.utils.util import wait_until
+
+
+class FeaturesTestBase(RedpandaTest):
+    """
+    Test cases defined in this parent class are executed as part
+    of subclasses that define node count below.
+    """
+    @cluster(num_nodes=3)
```

Comment:
would that return error if 2 out of 3 nodes are not used in `FeaturesTestSingleNode` ?

Reply:
Refactored this into two classes so that they only get as many nodes as they need.